code/game/objects/machinery/OpTable.dm

```diff
@@ -91,12 +91,17 @@
 	if(!buckling_human.equip_to_slot_or_del(new /obj/item/clothing/mask/breath/medical(buckling_human), SLOT_WEAR_MASK))
 		to_chat(user, span_danger("You can't fit the gas mask over their face!"))
 		return FALSE
-	buckling_human.internal = anes_tank
 	buckling_human.visible_message("[span_notice("[user] fits the mask over [buckling_human]'s face and turns on the anesthetic.")]'")
 	to_chat(buckling_human, span_information("You begin to feel sleepy."))
+	addtimer(CALLBACK(src, .proc/knock_out_buckled, buckling_human), rand(20, 40))
```

Comment:
why those numbers ?

Reply:
Just a rough 'a couple seconds' to simulate the knockout gas wearing off.